charts/stable/ghost/values.yaml

```diff
@@ -0,0 +1,58 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: ghost
+  # -- image tag
+  tag: 4.27.2
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+
+# See https://ghost.org/docs/config/#running-ghost-with-config-env-variables
+env:
+  url: "http://some-ghost.example.com"
+  database__client: mysql
+  database__connection__host: mariadb
+  database__connection__user: ghost
+  database__connection__password: ghost
+  database__connection__database: ghost
+  NODE_ENV: production
```

Comment:
Would it make sense to move this to the hardcoded section of the chart?

Reply:
No, it doesn't.

Reply:
Thank you for this elaborate comment. This really helps me understand the reasoning, and will get this PR approved só much quicker.

Reply:
Apologies for the sass in my previous comment, I may be a bit grumpy form work-stuff 😊
I notice that the line selection for my comment isn't really presented well in the GitHub UI. My comment was only relating to the `NODE_ENV` variable.

Reply:
That portion does make sense to add there, but Ghost dislikes that production env if the db is sqlite so I don't know which approach is the better one here.

Reply:
In that case I agree it's better to leave it configurable, since the other vars are configurable as well. 👍